src/app/navigation/Home.screen.tsx

```diff
@@ -15,6 +15,7 @@ export type HomeRouteParams = undefined;
 
 export const Home = () => {
   const [movies, setMovies] = useState([]);
+  const [nbMovies, setNbMovies] = useState<number>();
```

Comment:
No need for a derived state, you can just use `movies.length` to get the number of movies